runtime/autoload/netrw.vim

```diff
@@ -12523,14 +12543,19 @@ fun! s:ShowLink()
 " "  call Decho(printf("line#%4d: %s",line("."),getline(".")),'~'.expand("<slnum>"))
   if exists("b:netrw_curdir")
    norm! $?\a
-   let fname   = b:netrw_curdir.'/'.s:NetrwGetWord()
+   if exists("w:netrw_liststyle") && w:netrw_liststyle == s:TREELIST && exists("w:netrw_treetop")
+    let basedir = s:NetrwTreePath(w:netrw_treetop)
+   else
+    let basedir = b:netrw_curdir.'/'
+   endif
+   let fname = basedir .. s:NetrwGetWord()
```

Comment:
```suggestion
   let fname = basedir . s:NetrwGetWord()
```

Reply:
Yeah, I am mainly worried about consistency here. the lines around tend to use `.` for concatenation, so I'd prefer if we keep the style.
(or we can change the style per functions, but I would prefer to have this as a followup clean up PR then)